Q/qlever/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "qlever"
+version = v"0.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/ad-freiburg/qlever.git", "1d5503c65c604ce8fb5869da7f4f249a28be9dba"),
+]
+
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/qlever/
+
+git config --global advice.detachedHead false
+git submodule update --init --recursive
+
+mkdir build && cd build
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DUSE_PARALLEL=true -DABSL_PROPAGATE_CXX_STD=ON -DSTXXL_TRY_COMPILE_HEADERS=ON ..
+
+make -j $(nproc)
```

Comment:
I'm away from computer, but I seem to remember there is no `nproc` command
```suggestion
make -j ${nproc}
```

Reply:
@giordano Thanks!I copied this from the Yggdrasil codebase...is it worth/necessary to change the other instances as well?

- https://github.com/JuliaPackaging/Yggdrasil/blob/139747aa43faba7cc259e365a364d26af88c37e5/C/COSMA/build_tarballs.jl#L62
- https://github.com/JuliaPackaging/Yggdrasil/blob/139747aa43faba7cc259e365a364d26af88c37e5/F/faust/build_tarballs.jl#L70
- https://github.com/JuliaPackaging/Yggdrasil/blob/139747aa43faba7cc259e365a364d26af88c37e5/U/UBPF/build_tarballs.jl#L22
- https://github.com/jeremiahpslewis/Yggdrasil/blob/6c6f78cede7d3ca943dc58b937d1c001fb1343fa/U/Unicorn/build_tarballs.jl#L17
- https://github.com/jeremiahpslewis/Yggdrasil/blob/97ac9e8759c5d54355ae54d830a8a80628f5a132/X/XPA/build_tarballs.jl#L20